frontend/src/components/ui/config-details.ts

```diff
@@ -220,6 +220,15 @@ export class ConfigDetails extends LiteElement {
             msg("Block Ads by Domain"),
             crawlConfig?.config.blockAds
           )}
+          ${when(
```

Comment:
Could we render this all the time and present it as "Default" when unset?  Currently it doesn't appear when unset, meaning that when users look at the workflow settings page, (unless they are aware of this behavior) it is not obvious that the setting exists, or what its state is.

This is also why I'd like to change the label from "Custom User Agent" to just "User Agent", a "custom" one wouldn't make sense with "default" as the status, but hopefully without that adjective it makes sense!

Reply:
Yup, that's an easy and good change!

Also down for dropping the "Custom" :)

Reply:
And done!

Reply:
Screenshots/PR description also updated